sim/warrior/deep_wounds.go

```diff
@@ -66,16 +66,15 @@ func (warrior *Warrior) applyDeepWounds() {
 	}))
 }
 
+// Needs confirmation on classic functionality.  It is regular dot instead of a stacking one and if so does that mean it is only overwritten by a "higher" crit?
```

Comment:
Deep wound will be overwritten regardless if it's a lower or higher crit.

Reply:
Thank you, should be working then